posthog/models/person.py

```diff
@@ -41,32 +42,13 @@ def add_distinct_ids(self, distinct_ids: List[str]) -> None:
             self.add_distinct_id(distinct_id)
 
     def merge_people(self, people_to_merge: List["Person"]):
-        CohortPeople = apps.get_model(app_label="posthog", model_name="CohortPeople")
+        from posthog.api.capture import capture_internal
 
-        first_seen = self.created_at
-
-        # merge the properties
-        for other_person in people_to_merge:
-            self.properties = {**other_person.properties, **self.properties}
-            if other_person.created_at < first_seen:
-                # Keep the oldest created_at (i.e. the first time we've seen this person)
-                first_seen = other_person.created_at
-        self.created_at = first_seen
-        self.save()
-
-        # merge the distinct_ids
         for other_person in people_to_merge:
-            other_person_distinct_ids = PersonDistinctId.objects.filter(person=other_person, team_id=self.team_id)
-            for person_distinct_id in other_person_distinct_ids:
-                person_distinct_id.person = self
-                person_distinct_id.save()
-
-            other_person_cohort_ids = CohortPeople.objects.filter(person=other_person)
-            for person_cohort_id in other_person_cohort_ids:
-                person_cohort_id.person = self
-                person_cohort_id.save()
+            now = timezone.now()
+            event = {"event": "$create_alias", "properties": {"alias": other_person.distinct_ids[-1]}}
```

Comment:
Q: Why do we only create the alias event for the last distinct ID of the person? Shouldn't this take into account all the distinct ids?

Or is there an assumption that any distinct id works, since it should* uniquely map to a personID?

Reply:
That's the case here - plugin server will do a person_id lookup based on postgres and it will do the right thing based on that.

Reply:
Ah, right, even on CH instances, this all happens in Postgres, so we're safe. Cool.